packages/3id-did-resolver/package.json

```diff
@@ -45,5 +44,8 @@
     "transformIgnorePatterns": [
       "/node_modules(?!/did-jwt)/"
     ]
+  },
+  "dependencies": {
+    "did-resolver": "^1.1.0"
```

Comment:
Why was this moved to regular deps?

Reply:
https://github.com/ceramicnetwork/js-ceramic/blob/develop/packages/3id-did-resolver/src/index.ts#L1

Reply:
We only use the types so we shouldn't need to have it as a dep. What command are you using the `--production` flag on? And when do we need to do that? Usually we build before we publish to npm.

Reply:
Still confused by this.

Reply:
@oed  Without this, when the package is imported into `3box-js` and the example server is run, I get an error in the browser
```
Uncaught (in promise) TypeError: did_resolver_1.Resolver is not a constructor
    at new TileHandler (tileHandler.js:29)
    at new Ceramic (ceramic.js:39)
    at Function.eval (ceramic.js:59)
    at Generator.next (<anonymous>)
    at eval (ceramic.js:8)
    at new Promise (<anonymous>)
    at __awaiter (ceramic.js:4)
    at Function.create (ceramic.js:57)
    at Box._callee$ (3box.js:155)
    at tryCatch (runtime.js:45)
```

This is possibly because of the different versions of `did-resolver` used by 3box-js and ceramic. It might be only because I am volume mounting the package instead of using a published version, but since this was moved into dev dependencies since the last published version, it looked like a mistake. Can you please publish the latest version so we can test it?

Reply:
Nevermind, this error was being caused by the combination of lerna hoisting and docker volume mounting - just not going to hoist in local development.

Reply:
ok, I guess we can move it to here for now then. Let's merge this then make a new release!